### PR TITLE
Use raw format

### PR DESCRIPTION
**Pull request by ysksuzuki**
Signed-off-by: Yusuke Suzuki <yusuke-suzuki@cybozu.co.jp>

### CODE REVIEW

File: menu/cluster_yaml_generator.go [hunk 1]
```diff
@@ -231,10 +231,11 @@ func createWorkerNode(rack *rack, node *node, diskNumber int, spec *nodeSpec) *t
...
 	for i := 0; i < diskNumber; i++ {
 		nodeSpec.Volumes = append(nodeSpec.Volumes, types.NodeVolumeSpec{
-			Kind:  "raw",
-			Name:  fmt.Sprintf("data%d", i+1),
-			Size:  "50G",
-			Cache: "writeback",
+			Kind:   "raw",
+			Name:   fmt.Sprintf("data%d", i+1),
+			Size:   "50G",
+			Cache:  "writeback",
```
Comment (ymmt2005) [inline menu/cluster_yaml_generator.go:237]: I guess none can be more appropriate when the underlying storage is fast enough.

**Review by ymmt2005 — APPROVED** [review]
LGTM

**Comment by ysksuzuki** [comment]
The result of neco-apps test with raw format on my environment

```
Ran 60 of 60 Specs in 346.583 seconds
SUCCESS! -- 60 Passed | 0 Failed | 0 Pending | 0 Skipped


Ginkgo ran 1 suite in 5m49.649546304s
Test Suite Passed
```

**Comment by ysksuzuki** [comment]
Moco test

```
------------------------------
Test applications moco
  should work
  /home/yusuke/go/src/github.com/cybozu-go/neco-apps/test/moco_test.go:182

START: 2021-03-12T00:14:51Z
STEP: waiting mysqlcluster is ready
STEP: running kubectl moco mysql
END: 2021-03-12T00:15:00Z


• [SLOW TEST:9.579 seconds]
Test applications
/home/yusuke/go/src/github.com/cybozu-go/neco-apps/test/suite_test.go:40
  moco
  /home/yusuke/go/src/github.com/cybozu-go/neco-apps/test/suite_test.go:116
    should work
    /home/yusuke/go/src/github.com/cybozu-go/neco-apps/test/moco_test.go:182
------------------------------
```

**Comment by ymmt2005** [comment]
@ysksuzuki Great!

**Pull request merged**